frontend/src/component/project/Project/ProjectDoraMetrics/ProjectDoraFeedback/ProjectDoraFeedback.tsx

```diff
@@ -0,0 +1,158 @@
+                We are trying something experimental!
+            </StyledHeader>
+            <Typography>
+                We are considering adding project metrics to see how a project
+                performs. As a first step, we have added a lead time for changes
+                indicator that is calculated per feature toggle based on the
+                creation of the feature toggle and when it was first turned on
+                in an environment of type production.
+            </Typography>
+
+            <ConditionallyRender
+                condition={!sentObj.sent}
+                show={
+                    <StyledBtnContainer>
+                        {' '}
+                        <Typography>Is this useful to you?</Typography>
+                        <StyledBtnContainer>
+                            <Button
+                                sx={{ marginRight: '0.5rem' }}
+                                variant="contained"
+                                color="primary"
+                                onClick={() => onBtnClick('useful')}
+                            >
+                                Yes, I like the direction
+                            </Button>
+                            <Button
+                                variant="outlined"
+                                color="primary"
+                                onClick={() => onBtnClick('not useful')}
+                            >
+                                No, I don't see value in this
+                            </Button>
+                        </StyledBtnContainer>
+                    </StyledBtnContainer>
+                }
+                elseShow={
+                    <Typography sx={theme => ({ marginTop: theme.spacing(3) })}>
+                        Thank you for the feedback. Feel free to check out the
+                        sketches and leave comments, or get in touch with our UX
+                        team if you'd like to be involved in usertests and the
+                        development of this feature.
+                    </Typography>
+                }
+            />
+
+            <StyledDivider />
+            <StyledFlexBox>
+                <StyledFlexBox>
+                    <StyledIconWrapper>
+                        <PermMedia />
+                    </StyledIconWrapper>
+                    <StyledLink
+                        href="https://app.mural.co/t/unleash2757/m/unleash2757/1694006366166/fae4aa4f796de214bdb3ae2d5ce9de934b68fdfb?sender=u777a1f5633477c329eae3448"
+                        target="_blank"
+                        rel="noopener noreferer"
+                    >
+                        View sketches
+                    </StyledLink>
+                </StyledFlexBox>
+
+                <StyledFlexBox>
+                    <StyledIconWrapper>
+                        <Send />
+                    </StyledIconWrapper>
+                    <StyledLink href={mailtoURL}>
```

Comment:
This link does not work for me, but maybe its because I need to set default mail client. Not sure.